ui/pages/smart-transactions/smart-transaction-status-page/smart-transaction-status-page.tsx

```diff
@@ -0,0 +1,311 @@
+      </Text>
+    );
+  }
+
+  if (isSmartTransactionPending && isSmartTransactionTakingTooLong) {
+    title = t('smartTransactionTakingTooLong');
+    description = t('smartTransactionTakingTooLongDescription', [countdown]);
+    iconName = IconName.Clock;
+    iconColor = IconColor.primaryDefault;
+  } else if (isSmartTransactionPending) {
+    title = t('smartTransactionPending');
+    description = t('stxEstimatedCompletion', [countdown]);
+    iconName = IconName.Clock;
+    iconColor = IconColor.primaryDefault;
+  } else if (isSmartTransactionSuccess) {
+    title = t('smartTransactionSuccess');
+    iconName = IconName.Confirmation;
+    iconColor = IconColor.successDefault;
+  } else if (isSmartTransactionCancelled) {
+    title = t('smartTransactionCancelled');
+    description = t('smartTransactionCancelledDescription');
+    iconName = IconName.Danger;
+    iconColor = IconColor.errorDefault;
+  } else {
+    // E.g. reverted or unknown statuses.
+    title = t('smartTransactionError');
+    description = t('smartTransactionErrorDescription');
+    iconName = IconName.Danger;
+    iconColor = IconColor.errorDefault;
+  }
+
+  useEffect(() => {
+    let intervalId: NodeJS.Timeout;
+    if (isSmartTransactionPending) {
+      const calculateRemainingTime = () => {
+        const secondsAfterStxSubmission = smartTransaction?.creationTime
+          ? Math.round((Date.now() - smartTransaction.creationTime) / 1000)
+          : 0;
+        if (secondsAfterStxSubmission > stxDeadline) {
+          if (isSmartTransactionTakingTooLong) {
+            setTimeLeftForPendingStxInSec(0);
+            clearInterval(intervalId);
+            return;
+          }
+          setIsSmartTransactionTakingTooLong(true);
+        }
+        setTimeLeftForPendingStxInSec(stxDeadline - secondsAfterStxSubmission);
+      };
+      intervalId = setInterval(calculateRemainingTime, 1000);
+      calculateRemainingTime();
+    }
+    return () => clearInterval(intervalId);
+  }, [isSmartTransactionPending, isSmartTransactionTakingTooLong]);
+
+  useEffect(() => {
+    dispatch(hideLoadingIndication());
+  }, []);
+
+  return (
+    <Box
+      className="smart-transaction-status-page"
+      height={BlockSize.Full}
+      width={BlockSize.Full}
+      display={Display.Flex}
+      borderStyle={BorderStyle.none}
+      flexDirection={FlexDirection.Column}
+      alignItems={AlignItems.center}
+      marginBottom={0}
+    >
+      <Box
+        display={Display.Flex}
+        flexDirection={FlexDirection.Column}
+        alignItems={AlignItems.center}
+        justifyContent={JustifyContent.center}
+        paddingLeft={10}
+        paddingRight={10}
+        style={{ flexGrow: 1 }}
+      >
+        <Box
+          display={Display.Flex}
+          flexDirection={FlexDirection.Column}
+          alignItems={AlignItems.center}
+        >
+          <Box display={Display.Flex} style={{ fontSize: '48px' }}>
```

Comment:
Can we use a standard font size here?

Reply:
I'm using 48px here, because the largest IconSize is IconSize.Xl, which was below 48px and looked a bit small on the STX status page. Product wants 48px. I'm using IconSize.Inherit to achieve it. Other option could be to add a className and have it there.
```
<Box display={Display.Flex} style={{ fontSize: '48px' }}>
  <Icon
    name={iconName}
    color={iconColor}
    size={IconSize.Inherit}
    marginBottom={4}
  />
</Box>
```